RetrospectiveExtension.Frontend/scripts/setup_ci.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+##
+## This script installs the tools needed for CI
+##
+(
+  cd "$(dirname "$0")/.." || exit
+  set -euo pipefail
+
```

Comment:
Considering [Task #1499](https://dev.azure.com/csebostoncrew/Retrospectives-AzDO-Extension/_workitems/edit/1499), should we worry about locking npm version as well?

Reply:
There will be only one npm and node installed in the devContainer, and that is the npm in the `DockerFile` which will also be used here. What do you have in mind here?

Reply:
@phongcao @mhassaninmsft Let's hold this for a future PR - we can discuss at the next parking lot.

Reply:
Agreed